src/python_testing/TC_MCORE_FS_1_3.py

```diff
@@ -189,7 +189,7 @@ async def test_TC_MCORE_FS_1_3(self):
             device_node_id=th_server_th_node_id)
 
         # Wait for the device to appear on the DUT_FSA_BRIDGE.
-        await asyncio.sleep(2)
+        await asyncio.sleep(30)
```

Comment:
Please increase the timeout but only for the non-CI variant (self.is_ci).

Reply:
I commited a patch for this.